conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -29,6 +29,7 @@ CONDARC
 cat << EOF | docker run -i \
                         -v ${RECIPE_ROOT}:/recipe_root \
                         -v ${FEEDSTOCK_ROOT}:/feedstock_root \
```

Comment:
I don't think it is worth changing the mount point. It will just make things more complicated.